pkgs/development/python-modules/unstructured-client/default.nix

```diff
@@ -0,0 +1,72 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, wheel
+, certifi
+, charset-normalizer
+, dataclasses-json
+, deepdiff
+, idna
+, jsonpath-python
+, marshmallow
+, mypy-extensions
+, packaging
+, pypdf
+, python-dateutil
+, requests
+, six
+, typing-inspect
+, typing-extensions
+, urllib3
+}:
+
+buildPythonPackage rec {
+  pname = "unstructured-python-client";
+  version = "0.22.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "Unstructured-IO";
+    repo = "unstructured-python-client";
+    rev = "v${version}";
+    hash = "sha256-C3kpDD+EA1jJurZpGV0nVFqcwEOz4Daem8p2HKDR9gA=";
+  };
+
+  build-system = [
+    setuptools
+    wheel
+  ];
+
+  dependencies = [
+    certifi
+    charset-normalizer
+    dataclasses-json
+    deepdiff
+    idna
+    jsonpath-python
+    marshmallow
+    mypy-extensions
+    packaging
+    pypdf
+    python-dateutil
+    requests
+    six
+    typing-inspect
+    typing-extensions
+    urllib3
+  ];
+
+  # tests do network access
```

Comment:
All of them or can we disable select test cases?

Reply:
50% of the tests cases fail. I don't think it makes sense for the long term to keep disabling single tests, it will cause to much maintenance burden if they change tests.
I've added a comment.